api/base/serializers.py

```diff
@@ -79,6 +79,7 @@ def _tpl(val):
     return None
 
 
+#todo probably needs refactoring
```

Comment:
Forgot about this.  Left from an older commit.  Will change when I remove the other file